### PR TITLE
first implementation

### PR DESCRIPTION
**Pull request by itajaja**
it's very opinionated and of dubious usefullness, but I got it locally, so I thought I could share it

### CODE REVIEW

File: {{cookiecutter.app_name}}/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+COPY requirements.txt  /usr/src/app/
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+EXPOSE 8000
+
+ENV PYTHONUNBUFFERED=1
+ENV NEW_RELIC_CONFIG_FILE=newrelic.ini
```
Comment (cancan101) [inline {{cookiecutter.app_name}}/Dockerfile:15]: why not make these one layer?

Comment (itajaja) [reply {{cookiecutter.app_name}}/Dockerfile:15]: no strong reason. that's how it is in some of our repos

File: {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+from flask import Flask
+
+from .config import config
+
+# -----------------------------------------------------------------------------
+
+app = Flask('{{cookiecutter.app_name}}')
+app.config.from_mapping(config)
+
+if not app.debug:
```
Comment (cancan101) [inline {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/__init__.py:11]: perhaps put a comment here as to why

File: {{cookiecutter.app_name}}/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt  /usr/src/app/
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+EXPOSE 8000
```
Comment (cancan101) [inline {{cookiecutter.app_name}}/Dockerfile:12]: 8000 or 5000?

File: {{cookiecutter.app_name}}/Dockerfile (hunk 1 above)
Comment (itajaja) [reply {{cookiecutter.app_name}}/Dockerfile:15]: see also https://github.com/nodejs/docker-node/blob/581eebd097343c9f1c1ceb5260cd2ec770410e29/7.10/Dockerfile or https://github.com/docker-library/python/blob/cd1f11aa745a05ddf6329678d5b12a097084681b/2.7/Dockerfile (just two random official images I searched)

**Comment by chirag04** [comment]
@itajaja add `cookiecutter create view|route --schema --model` or something like that!

parse the view file, generate ast or similar, append the new view, sort, pretty print everything.

File: {{cookiecutter.app_name}}/Dockerfile (hunk 3 above)
Comment (itajaja) [reply {{cookiecutter.app_name}}/Dockerfile:12]: 8000 :)

File: {{cookiecutter.app_name}}/Dockerfile (hunk 1 above)
Comment (cancan101) [reply {{cookiecutter.app_name}}/Dockerfile:15]: https://www.spantree.net/blog/2015/04/26/tips-docker.html

Comment (itajaja) [reply {{cookiecutter.app_name}}/Dockerfile:15]: do you trust a random guy on the internet or official docker images? ⚖️

**Comment by itajaja** [comment]
sure @chirag04, you wanna do it :P

File: {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/__init__.py (hunk 2 above)
Comment (itajaja) [reply {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/__init__.py:11]: sincerely I cargo culted this. @taion?

File: {{cookiecutter.app_name}}/Dockerfile (hunk 1 above)
Comment (cancan101) [inline {{cookiecutter.app_name}}/Dockerfile]: THIS GUY!!! https://github.com/ButterflyNetwork/olympus-storage/blob/247eecf7779efd1035d9762d28fbecba5a3fab3a/Dockerfile#L24-L25

**Pull request merged**